app/src/main/res/values/strings.xml

```diff
@@ -531,6 +531,9 @@
     <string name="error_sharing_cover">Error sharing cover</string>
     <string name="custom_manga_info">Custom manga info</string>
     <string name="set_as_default">Set as default</string>
+    <string name="filter_groups">Filter scanlator groups</string>
+    <string name="filter_groups_title">Scanlator groups to show</string>
```

Comment:
oh right we can also remove this string and just use filter_groups as the dialog title

Reply:
I honestly like having a title as "Scanlator groups to hide" but have removed it